src/main/java/com/axlsreborn/CsvWriter.java

```diff
@@ -8,35 +8,36 @@
 
 public class CsvWriter {
 
-    public static StringBuffer csvSteamAchievement(SteamAchievement steamAchievement, char csvDelimiter) {
+    public static String toCsvString(SteamAchievement steamAchievement, char csvDelimiter) {
         StringBuffer stringBuffer = new StringBuffer();
 
-        return stringBuffer
+        stringBuffer
                 .append("\"")
                 .append(steamAchievement.getAchievementName())
-                .append("\"")
+                .append("\" ")
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementPercent())
-                .append("\"")
+                .append("\" ")
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementDescription())
-                .append("\"")
+                .append("\" ")
                 .append(csvDelimiter)
-                .append("\"")
+                .append(" \"")
                 .append(steamAchievement.getAchievementIconUrl())
                 .append("\"")
                 .append("\n");
```

Comment:
It's been a while since I've used a string buffer, but couldn't this be 

```suggestion
                .append("\"\n");
```